thinc/util.py

```diff
@@ -203,12 +203,20 @@ def copy_array(dst: ArrayXd, src: ArrayXd) -> None:  # pragma: no cover
         numpy.copyto(dst, src)  # type: ignore
 
 
-def to_categorical(Y: IntsXd, n_classes: Optional[int] = None) -> FloatsXd:
+def to_categorical(
+    Y: IntsXd,
+    n_classes: Optional[int] = None,
```

Comment:
I think we should make `n_classes` and `label_smoothing` keyword-only arguments. Unfortunately this would be breaking for `n_classes`, but we could do it at least for `label_smoothing` already.